var/spack/repos/builtin/packages/pango/package.py

```diff
@@ -31,6 +31,7 @@ class Pango(AutotoolsPackage):
     depends_on("libxft", when='+X')
     depends_on("glib")
     depends_on('gobject-introspection')
+    depends_on('libffi')
```

Comment:
How did you discover that adding the `libffi` dependency allowed it to pick up `glib`?

Reply:
@adamjstewart See the `config.log` posted in #12383. Search for the string "Package libffi was not found in the pkg-config search path."